connector/connect/common/src/main/protobuf/spark/connect/commands.proto

```diff
@@ -298,6 +309,18 @@ message StreamingQueryCommandResult {
     // Logical and physical plans as string
     string result = 1;
   }
+
+  message ExceptionResult {
+    // Exception message as string
+    oneof exception_content {
+      bool no_exception = 1;
```

Comment:
why need to have this? Can we just not set the exception field if there is no exception?